static/src/javascripts/projects/common/modules/experiments/tests/contributions-countries-uk.js

```diff
@@ -0,0 +1,143 @@
+define([
+    'bean',
+    'qwery',
+    'common/utils/$',
+    'common/utils/template',
+    'common/views/svg',
+    'common/utils/fastdom-promise',
+    'common/utils/mediator',
+    'text!common/views/contributions-embed.html',
+    'common/utils/robust',
+    'inlineSvg!svgs/icon/arrow-right',
+    'common/utils/config',
+    'common/utils/cookies',
+    'common/modules/experiments/embed',
+    'common/utils/ajax',
+    'common/modules/commercial/commercial-features'
+
+], function (bean,
+             qwery,
+             $,
+             template,
+             svg,
+             fastdom,
+             mediator,
+             contributionsEmbed,
+             robust,
+             arrowRight,
+             config,
+             cookies,
+             embed,
+             ajax,
+             commercialFeatures
+) {
+
+
+    return function () {
+
+        this.id = 'ContributionsCountriesUk';
+        this.start = '2016-10-28';
+        this.expiry = '2016-11-04';
+        this.author = 'Phil Wills';
+        this.description = 'Test whether different messages perform better/worse in different countries (UK)';
+        this.showForSensitive = false;
+        this.audience = 0.15;
+        this.audienceOffset = 0.56;
```

Comment:
Sorry, should be clear that we've done much less testing in the US, so that's why we haven't already used up to 0.56 there.